src/badge/lib/badge.scss

```diff
@@ -0,0 +1,34 @@
+$_tbds-badge-font-size: 0.8em !default;
+$_tbds-badge-background-color: #f5f5f5 !default;
+$_tbds-badge-color: $tbds-brand-gray !default;
+
+.tbds-badge {
+  background-color: $_tbds-badge-background-color;
+  border: 0;
+  border-radius: $_tbds-badge-font-size / 4;
+  color: $_tbds-badge-color;
+  display: inline-block;
+  font-size: $_tbds-badge-font-size;
+  font-weight: normal;
+  line-height: 1;
+  padding: 0.25em 0.5em;
+  vertical-align: baseline;
+  white-space: nowrap;
+}
+
+a.tbds-badge,
```

Comment:
It would be nice to remove these qualified selectors. I’m curious to hear your thoughts on the design intention here. 

We [don’t currently remove `text-decoration` on `hover` and `focus` of anchors on the global level](https://github.com/thoughtbot/design-system/blob/a711610c6dcb0967d9d7d75aeeaad0937044870b/src/elements/lib/typography.scss#L14-L27); should we do that? I think that would remove the need for these selectors…

Reply:
I like qualified selectors when it's tied to imperative functionality, so being able to have interactivity on a component snap in by virtue of declaring the right element. The converse is being able to grant a non-interactive element interactive behavior via assigning it a class like `tbds-badge--link`, which can lead to abuse. 

As for the text decoration stuff, I was ensuring that state change wasn't relying on color alone. What do you think of moving that up into global scope?

Reply:
> What do you think of moving that up into global scope?

I think that sounds like a great idea - perhaps a different PR?

Reply:
Will do!